source/configure/configuration-settings.rst

```diff
@@ -1770,15 +1770,15 @@ To ensure that users can contact you for assistance, set this value to an email
 | This feature's ``config.json`` setting is ``"SupportEmail": ""`` with string input.                                                                                  |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
-Terms of Service link
+Terms of Use link
 ^^^^^^^^^^^^^^^^^^^^^^
 
-Configurable link to Terms of Service your organization may provide to end users on the footer of the sign-up and login pages. By default, links to a Terms of Service page hosted on about.mattermost.com. If changing the link to a different Terms of Service, make sure to include the "Mattermost Conditions of Use" notice to end users that must also be shown to users from the "Terms of Service" link.
+Configurable link to Terms of Use your organization may provide to end users on the footer of the sign-up and login pages. By default, links to a Terms of Use page hosted on mattermost.com/terms-of-service/. If changing the link to a different Terms of Use, make sure to include the "Mattermost Terms of Use" notice to end users that must also be shown to users from the "Terms of Use" link.
```

Comment:
Given the change in terminology from "terms of service" to "terms of use", is there value in updating the website page URL as well to ``mattermost.com/terms-of-use``? 

@jamiehurewitz - If you agree, would you like me to follow up with Marketing to propose this change?

Reply:
Sure. Makes sense to me.